include/mc_tasks/lipm_stabilizer/Contact.h

```diff
@@ -228,6 +235,8 @@ struct MC_TASKS_DLLAPI Contact
   double friction_ = 0.7; /**< Friction coefficient for the contact surface */
 
   Eigen::Matrix<double, 16, 6> wrenchFaceMatrix_; /**< Matrix of single-contact wrench cone inequalities */
+  Eigen::Matrix<double, 12, 6>
+      wrenchFaceMatrixCoPFree_; /**< Matrix of single-contact wrench cone inequalities without CoP constraints */
```

Comment:
I think it would be better to stick to a single `wrenchFaceMatrix_` but move the CoP geometric constraint to the last lines of the matrix so that the block operation can be done when building the problem